### PR TITLE
feat: update the colours from the variable, update the introduction l…

### PR DESCRIPTION
**Pull request by isaac-shek**
#### 📲 What

- Switch from Amido to Ensono brand - buttons, colours.
- Split the sections into React components and add the css to their specific components.
- Fixed the missing images
- Layout has changed for some of the sections
- Added Underscore to the files and directories to prevent them from creating website routes
- Added PlusJakartaSans and Inter fonts, font sizes into the project

#### 🤔 Why
		
Why it's needed, background context.
		
#### 🛠 How
		
More in-depth discussion of the change or implementation.

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR.
		 
#### 🕵️ How to test

Notes on how a reviewer can test the changes, e.g. how to run the tests.

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: src/pages/_Components/EnsonoStacksImage/EnsonoStacksImage.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    const baseUrl = useBaseUrl("docs/");
+
+    return (
+        <div className={customStyle.ensonoStacksImageContainer}>
+            <div className={customStyle.wrapperImage}>
+                <img src={useBaseUrl('img/ensonoImage.png')} alt="Ensono Image"/>
+            </div>
+            <div className={customStyle.wrapper}>
+                <AmidoStacksLogo/>
+                {/* <h1>{text.title}</h1> */}
```
Comment (leesaxby) [inline src/pages/_Components/EnsonoStacksImage/EnsonoStacksImage.tsx:24]: Do we want to remove this commented code?

File: src/pages/_Components/EnsonoStacksImage/EnsonoStacksImage.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                <p>{text.description}</p>
+                <div style={{ marginTop: 20 }}>
```
Comment (leesaxby) [inline src/pages/_Components/EnsonoStacksImage/EnsonoStacksImage.tsx:26]: Would we want to move this style into a `css` file?

File: src/pages/_Components/WhatStacksCanDo/WhatStacksCanDo.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+const text = {
+    clientFeedbackMessage: '"Ensono Stacks made our transformation significantly easier and gave us confidence right from day one" - '
+}
+
+const WhatStacksCanDo = () => {
+    const baseUrl = useBaseUrl("docs/");
+
+    return (
+        <div>
+            <span className={styles.primaryColour}><i style={{display: 'inline-block'}} className={styles.textCenter}>{text.clientFeedbackMessage}<strong>Ensono Client</strong></i></span>
```
Comment (leesaxby) [inline src/pages/_Components/WhatStacksCanDo/WhatStacksCanDo.tsx:17]: Do we want to pull `style={{display: 'inline-block'}}` out into a css file?

File: src/pages/_Components/TechSupportedByStacks/TechSupportedByStacks.tsx [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+      };
+
+    return (
+        <div className={customStyle.mainHolder}>
+            <div className={customStyle.main}>
+                <section className={styles.section}>
+                    <h2 className={styles.textCenter}>Technologies supported by Stacks</h2>
+                    <div className={customStyle.tableContainer}>
+                        <div className={clsx(customStyle.flexTable)} role="rowGroup">
+                        <div><div><CSharpLogo className={clsx(customStyle.flexRow)} role="cell" onClick={() => triggerSearch("C#")} /></div></div>
```
[the remark this answers is not in the document]
Comment (isaac-shek) [reply src/pages/_Components/TechSupportedByStacks/TechSupportedByStacks.tsx:67]: This was copied from index.js. I do want to refactor this file but this refactoring could be big.

[the remark this answers is not in the document]
Comment (isaac-shek) [reply src/pages/_Components/TechSupportedByStacks/TechSupportedByStacks.tsx:67]: Might be worth having a separate PR to refactor this.

File: src/pages/_Components/Picker/Picker.tsx [hunk 5]
```diff
@@ -0,0 +1,122 @@
+import React, {useState} from "react";
```
Comment (leesaxby) [inline src/pages/_Components/Picker/Picker.tsx:1]: Think this file could possibly do with a refactor, particularly around the optionMapper data structure.
But if we're eager to get this in we could look at that later.

Comment (isaac-shek) [reply src/pages/_Components/Picker/Picker.tsx:1]: Will refactor Picker and TechSupportedByStacks components separate from the PR

File: src/pages/_Components/EnsonoStacksImage/EnsonoStacksImage.tsx (hunk 1 above)
Comment (isaac-shek) [reply src/pages/_Components/EnsonoStacksImage/EnsonoStacksImage.tsx]: done

File: src/pages/_Components/EnsonoStacksImage/EnsonoStacksImage.tsx (hunk 2 above)
Comment (isaac-shek) [reply src/pages/_Components/EnsonoStacksImage/EnsonoStacksImage.tsx]: done

File: src/pages/_Components/TechSupportedByStacks/TechSupportedByStacks.module.css [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+.supportedTechs {
+    color: var(--ifm-heading-color);
+    text-align: center;
+    padding-top: 30px;
+    margin-top: 30px;
+    margin-bottom: 30px;
+  }
+  
+  .supportedTechs h3 {
+    font-size: max(28px, min(3vw, 44px));
```
[the remark this answers is not in the document]
Comment (isaac-shek) [reply src/pages/_Components/TechSupportedByStacks/TechSupportedByStacks.module.css]: done

File: src/pages/_Components/TechSupportedByStacks/TechSupportedByStacks.module.css [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+    font-weight: 700;
+    margin-bottom: 60px;
+  }
+  
+  .supportedTechs p {
+    font-size: max(18px, min(2vw, 30px));
```
[the remark this answers is not in the document]
Comment (isaac-shek) [reply src/pages/_Components/TechSupportedByStacks/TechSupportedByStacks.module.css]: removed this, not used

File: src/pages/_Components/WhatStacksCanDo/WhatStacksCanDo.tsx (hunk 3 above)
Comment (isaac-shek) [reply src/pages/_Components/WhatStacksCanDo/WhatStacksCanDo.tsx]: done

**Review by leesaxby — APPROVED** [review]

**Pull request merged**